src/uct/ib/base/ib_device.c

```diff
@@ -512,11 +512,17 @@ const char *uct_ib_roce_version_str(int roce_ver)
     case UCT_IB_DEVICE_ROCE_V2:
         return "RoCE v2";
     default:
-        ucs_error("invalid RoCE version: %d", roce_ver);
-        return "unknown";
+        return "";
```

Comment:
need to return `"<unknown roce version>"` (sorry looks like github filtered out the brackets from my previous comment)